build/unicode_test.go

```diff
@@ -0,0 +1,98 @@
+package build
+
+import (
+	"testing"
+)
+
+func TestAppendRune(t *testing.T) {
+	table := &CodeTable{CodeRange{48, 57}}
+
+	table.AppendRune(45)
+	CompareTable(t, *table, "09--", false)
+
+	table.AppendRune(44)
+	CompareTable(t, *table, "09--,,", false)
+
+	table.AppendRune(95)
+	CompareTable(t, *table, "09--,,__", false)
+}
```

Comment:
Agree, using rune literals will make this easier to read. Happy Go supports UTF-8 in source.